### PR TITLE
843228 - Maui Scheduler and DataForm content changes added

### PR DESCRIPTION
**Pull request by Kerubananthan3761**
## Description
Maui scheduler data form content changes added

### CODE REVIEW

File: MAUI/DataForm/layout.md [hunk 1]
```diff
@@ -398,4 +398,27 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
-Here, the `Name` field will be hidden.
\ No newline at end of file
+Here, the `Name` field will be hidden.
+
+## Change the editor height
+
+The data form editor height is changed by using the `EditorHeight` property of the [DataFormItem](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataForm.DataFormItem.html).
```
Comment (Jeyasri-Murugan) [inline MAUI/DataForm/layout.md:405]: The .NET MAUI DataForm allows you to modify the height of each editor to fit the height of the data form or to customize the height based on text size by using the `EditorHeight` property of

File: MAUI/DataForm/layout.md [hunk 2]
```diff
@@ -398,4 +398,27 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
...
+
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
```
Comment (Jeyasri-Murugan) [inline MAUI/DataForm/layout.md:411]: Remove from upcoming feature
Support for DataFormViewItem’s auto height.

https://help.syncfusion.com/maui/dataform/migration#upcoming-features

File: MAUI/DataForm/Editors.md [hunk 3]
```diff
@@ -373,6 +373,36 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
...
+            amount.AllowNull = true;
+            amount.UpDownPlacementMode = NumericEditorUpDownPlacementMode.Inline;
+        }
+    }
+}
+
+{% endhighlight %}
+{% endtabs %}
+
 ## Masked text editor
```
Comment (Jeyasri-Murugan) [inline MAUI/DataForm/Editors.md:406]: Update page for missed editors also
https://help.syncfusion.com/maui/dataform/getting-started#defining-editors

Numeric

File: MAUI/Scheduler/migration.md [hunk 4]
```diff
@@ -975,6 +975,12 @@ this.Content = scheduler;
 <td>{{'[ResourceIds]()'| markdownify }}</td>
 <td>Gets or sets the resource ids property for the schedule appointment to allocate the corresponding resource to the appointment.</td></tr>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:976]: Update in this table for TextColor and check and update any other places also 
https://help.syncfusion.com/maui/scheduler/appointments#scheduler-appointment-source-mapping

File: MAUI/Scheduler/migration.md [hunk 5]
```diff
@@ -975,6 +975,12 @@ this.Content = scheduler;
...
 
+<tr>    
+<td>{{'[TextColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.ScheduleAppointment.html#Syncfusion_SfSchedule_XForms_ScheduleAppointment_TextColor)'| markdownify }}</td>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:979]: update TextColormapping in all examples
https://help.syncfusion.com/maui/scheduler/appointments#creating-business-objects

File: MAUI/Scheduler/migration.md [hunk 6]
```diff
@@ -975,6 +975,12 @@ this.Content = scheduler;
...
+<td>TextColor</td>
+<td>Gets or sets the text color for an appointment in the SfScheduler.</td></tr>
```
Comment (Jeyasri-Murugan) [inline MAUI/Scheduler/migration.md:981]: Remove
Support to change text color for each scheduler appointment.

https://help.syncfusion.com/maui/scheduler/migration#upcoming-features

File: MAUI/DataForm/layout.md (hunk 1 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/layout.md]: content changed

File: MAUI/DataForm/layout.md (hunk 2 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/layout.md:411]: removed

File: MAUI/DataForm/Editors.md (hunk 3 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/Editors.md:406]: updated for missed editors

File: MAUI/DataForm/layout.md [hunk 7]
```diff
@@ -398,4 +398,27 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
...
+private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+{
+    if (e.DataFormItem != null)
+    {
```
[the remark this answers is not in the document]
Comment (Kerubananthan3761) [reply MAUI/DataForm/layout.md:415]: Remarks added

File: MAUI/Scheduler/migration.md (hunk 4 above)
Comment (Kerubananthan3761) [reply MAUI/Scheduler/migration.md]: Updated TextColor

File: MAUI/Scheduler/migration.md [hunk 8]
```diff
@@ -1190,6 +1190,11 @@ this.Content = scheduler;
 <td>{{'[ResourceIdsMapping]()'| markdownify }} (From DaysView, TimelineView)</td>
 <td>Gets or sets the resource ids mapping to map the custom appointment property with the schedule appointments resource ids property.</td></tr>
 
+<tr>
+<td>{{'[TextColorMapping](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfSchedule.XForms.ScheduleAppointmentMapping.html#Syncfusion_SfSchedule_XForms_ScheduleAppointmentMapping_TextColorMapping)'| markdownify }}</td>
+<td>TextColorMapping</td>
+<td>Gets or sets the TextColor Mapping to map the custom appointment property with schedule appointments text color property.</td></tr>
```
Comment (florenceliliana) [inline MAUI/Scheduler/migration.md:1196]: <td>Gets or sets the TextColor Mapping to map the custom appointment property with the scheduled appointment's text color property.</td></tr>

File: MAUI/DataForm/Editors.md [hunk 9]
```diff
@@ -373,6 +373,36 @@ private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs
 {% endhighlight %}
 {% endtabs %}
 
+#### Numeric up-down editor
+
+The numeric up-down button will be displayed in the numeric editor, When `UpDownPlacementMode` property of `DataFormNumericItem` is set to `NumericEditorUpDownPlacementMode.Inline`. By defult, the `NumericEditorUpDownPlacementMode` is `Hidden`.
```
Comment (florenceliliana) [inline MAUI/DataForm/Editors.md:378]: The numeric up-down button will be displayed in the numeric editor when the `UpDownPlacementMode` property of `DataFormNumericItem` is set to `NumericEditorUpDownPlacementMode.Inline`. By default, the `NumericEditorUpDownPlacementMode` is `Hidden`.

**Review by florenceliliana — CHANGES_REQUESTED** [review]
I have completed the content review.

File: MAUI/DataForm/Editors.md (hunk 9 above)
Comment (Kerubananthan3761) [reply MAUI/DataForm/Editors.md]: content changed

File: MAUI/Scheduler/migration.md (hunk 8 above)
Comment (Kerubananthan3761) [reply MAUI/Scheduler/migration.md]: content changed

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**